lib/DraggableCore.js

```diff
@@ -229,6 +229,10 @@ export default class DraggableCore extends React.Component<DraggableCoreProps, D
   }
 
   handleDragStart: EventHandler<MouseTouchEvent> = (e) => {
+
+    // Disable touch scrolling while dragging on mobile devices.
+    document.ontouchmove = function(e){ e.preventDefault(); };
```

Comment:
This makes an assumption that the developer isn't using `document.ontouchmove` for something else. Can this be done via addeventListener() instead?

Reply:
```js
window.addEventListener('touchstart', function(e) {
e.preventDefault();
}, { passive: false } );
```